Source/Atj/data/mvp_scenario.json

```diff
@@ -41,6 +41,20 @@
         "hacky_sack_club_flyer_1"
     ],
     "triggers": [
+        {
+            "name": "zach_mood_1_trigger",
+            "all_of": [
+                {
+                    "type": "npc_mood_check",
+                    "npc": "sara",
+                    "check": "LTorEQ",
```

Comment:
Per the newly updated [mood eng doc](https://docs.google.com/document/d/1o2DTrFQRu-fy56q73oHPsm1Jky4IeuaGDioQ8jUGDR8/edit#heading=h.j3tni75c59nb), you could use `EQ` here and a value of `1`.

Reply:
I want this trigger to occur when sara hits a mood threshold of 2. So I need to use a value of `2` not `1` right?

Reply:
Oh, I misunderstood. Yes, you can dismiss my comment.